### ISSUE TITLE
i try to use it, but it's raw

### ISSUE THREAD

**Issue by hbakhtiyor** [hyperledger-cello/cello#577]
something is not working, https://github.com/hyperledger-labs/fabric-operations-console is better than cello

**Comment by yeasy** [comment]
What thing is not working? Assume you're running with the code at the main branch.

**Comment by hbakhtiyor** [comment]
yes, using main branch, running well, but when to create anything, getting 400 error

**Comment by yeasy** [comment]
@fengyangsy will help have a look. Please input more steps to reproduce, thanks!

**Comment by hbakhtiyor** [comment]
<img width="1849" alt="Screenshot 2023-11-06 at 10 39 52 AM" src="https://github.com/hyperledger/cello/assets/3434496/2a9ee0a3-0c8b-43e4-ac6d-82520125b5cc">
<img width="2560" alt="Screenshot 2023-11-06 at 10 34 16 AM (2)" src="https://github.com/hyperledger/cello/assets/3434496/94dc6a13-8ef5-491d-9b27-c31d16f5688a">

all tabs give errors

**Comment by fengyangsy** [comment]
1、@xichen1 Please help check if there have been any changes to the chaincode page.
2、Cello is a distributed deployment system, and each Cello should belong to only one organization, so after logging in, there is no need to create other organizations. We will remove the new button.

**Comment by hbakhtiyor** [comment]
all tabs have 400 error, not only chaincode

**Comment by hbakhtiyor** [comment]
>  and each Cello should belong to only one organization,
how to set up?

**Comment by MyungJH** [comment]
Hello all.

I have a same problem. 400 error
Creating the node was successful, but ca's user registration failed.
When registering a user, the message "Success" appears, but at the same time, an error message appears.
![image](https://github.com/hyperledger/cello/assets/114722563/87a92e43-56f8-4184-b52f-3e95ccc89912)

and also failed make channel.
and show up this error ↓
请求错误 400: http://localhost:8081/api/v1/channels
发出的请求有错误，服务器没有进行新建或修改数据的操作。

**Comment by Linkadi98** [comment]
@MyungJH I have faced this error too but the code in main branch is broken for now. Well don't know when the authors can fix some errors like you pointed out.
